src/sentry/static/sentry/app/utils/projects.tsx

```diff
@@ -320,15 +419,12 @@ async function fetchProjects(
     query.all_projects = 1;
   }
 
-  let hasMore = false;
-  let nextCursor = null;
-  const [results, _, xhr] = await api.requestPromise(
-    `/organizations/${orgId}/projects/`,
-    {
-      includeAllArgs: true,
-      query,
-    }
-  );
+  let hasMore: null | boolean = false;
```

Comment:
@billyvg Probably doesn't matter but I wonder if it would be better to coerce `null` into `false`